Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -129,6 +129,8 @@ class ShopperInsightsViewController: PaymentButtonBaseViewController {
         button.isEnabled = false
         
         let paypalRequest = BTPayPalVaultRequest()
+        paypalRequest.userAuthenticationEmail = emailView.textField.text ?? nil
+        
```

Comment:
Is this a property a merchant has to populate from their UI or however they get the email?

Reply:
Yes!